tests/functional/test_confirm_department.py

```diff
@@ -1,28 +1,34 @@
+import pytest
 from flask import url_for
```

Comment:
Imports should be in alphabetical order within the groupings of core, 3rd party and local modules.